package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-utils-formattingutils",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
just making sure we are aware that this is a Major version update meaning that users using ^1.0.0 will not be able to use it going forward...